modules/apps/commerce/commerce-frontend-js/test/components/AddToCart/WrapperWithQuantity.js

```diff
@@ -0,0 +1,260 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import {act, cleanup, fireEvent, render, wait} from '@testing-library/react';
+import React from 'react';
+
+import WrapperWithQuantity from '../../../src/main/resources/META-INF/resources/components/add_to_cart/WrapperWithQuantity';
+
+describe('WrapperWithQuantity', () => {
+	const AddToCartButton = (props) => {
+		return <button data-test-quantity={props.quantity} />;
+	};
+
+	const INITIAL_PROPS = {
+		AddToCartButton,
+		channel: {
+			currencyCode: 'USD',
+			id: 12345,
+		},
+		cpInstance: {
+			skuId: 54321,
+		},
+		settings: {
+			withQuantity: {
+				forceDropdown: true,
+			},
+		},
+	};
+
+	describe('by interaction', () => {
+		let Component;
+
+		beforeEach(() => {
+			jest.resetAllMocks();
+			jest.useFakeTimers();
+		});
+
+		afterEach(() => {
+			cleanup();
+
+			Component = null;
```

Comment:
Same as my comments from the other test. No need to keep this value around for each test.